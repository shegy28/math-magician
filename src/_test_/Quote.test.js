import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Quote from '../components/Quote';
import '@testing-library/jest-dom';

test('renders /Mathematics is not about numbers...', () => {
  render(<Quote />);
  const linkElement = screen.getByText(/Mathematics is not about numbers, equations, computations, or algorithms:/i);
  expect(linkElement).toBeInTheDocument();
});

test('expect home component to match snapshot', () => {
  const view = renderer.create(<Quote />).toJSON();
  expect(view).toMatchSnapshot();
});
