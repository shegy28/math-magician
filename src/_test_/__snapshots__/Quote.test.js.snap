// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect home component to match snapshot 1`] = `
<p
  className="quotes"
>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -William Paul Thurston
</p>
`;
